server.js

```diff
@@ -2,4 +2,12 @@
 
 const unleash = require('./lib/server-impl');
 
-unleash.start();
+unleash
```

Comment:
This seems like a beautiful quality of life feature that could be delivered outside of this PR, and help immediately. Not sure if console.log or some form of more generic logger interface.